<launch>
    <rosparam command="load" file="$(find usv_mission_planner)/config/mission_planner.yaml" />
    <rosparam command="load" file="$(find usv_map)/config/map_service.yaml" />
    <rosparam command="load" file="$(find usv_map)/config/map_preprocessing.yaml" />
    <node pkg="usv_mission_planner"
        name="test_astar"
        type="test_astar"
        respawn="false"
        output="screen"
        launch-prefix="">
    </node>
</launch>